public/main.js

```diff
@@ -40,15 +51,35 @@ $jobSearch.addEventListener('submit', () => {
 $createUser.addEventListener('submit', () => {
   event.preventDefault()
   const formData = new FormData($createUser)
-  post('/users', formData)
+  if (window.location.hash === '#profile/create') {
```

Comment:
Use a hidden input field in your form that you populate when you're editing a profile. Then when you generate your request body, you'll know if it's a `PUT` or `POST` based on the presence of an `id.

Push the details of this logic into a general `save` function.